default.nix

```diff
@@ -11,7 +11,7 @@ let
 
   yarn2nix = pkgs.stdenv.mkDerivation {
     name = "yarn2nix";
-    src = pkgs.nix-gitignore.gitignoreSource [ ".git/" ] ./yarn2nix;
+    src = "${pkgs.nix-gitignore.gitignoreSource [ ".git/" ] ./.}/yarn2nix";
```

Comment:
are you sure this works correctly? It diverges from how cabal sees things, no? e.g. it will allow to physically break out of the directory with `..`, which might break the cabal package without us noticing.

Reply:
This derivation only copies stuff from `static` to the outputs and runs the clean packaging stuff, so no sandboxing issue I'd say. Not sure though if the clean packaging stuff has any merit because it doesn't really relate to the `minimalHaskellSource the package is actually built with…

Reply:
> Not sure though if the clean packaging stuff has any merit because it doesn't really relate to the `minimalHaskellSource the package is actually built with…

yeah, I’ve thought about it, but didn’t want to invest any extra thoughts.